airflow/dags/payments_loader/preprocess_columns.py

```diff
@@ -20,23 +20,23 @@
 # However, if we move to processing only the data for a given day each task,
 # we should change the SRC_DIR glob to use a datetime
 
-SRC_DIR = "gs://littlepay-data-extract-prod/mst/{table_name}/*.psv"
-STAGE_DIR = "mst/{table_name}"
-DST_DIR = "mst/processed/{table_name}"
-
-
-def main(execution_date, **kwargs):
-
+DEFAULT_SRC_URL_TEMPLATE = (
+    "gs://littlepay-data-extract-prod/{provider_name}/"
+    "{provider_name}/{table_name}/*.psv"
+)
+DEFAULT_STG_DIR_TEMPLATE = "{table_name}"
+DEFAULT_DST_DIR_TEMPLATE = "processed/{table_name}"
```

Comment:
Can we change the top-level folder from processed to `littlepay-processed`, and the staging dir to `littlepay-staging/{table_name}`? We'll have an `rt-processed` folder for realtime, so this should keep everything matched up!

edit: (or could see using `payments-processed` instead)

Reply:
That sounds good. I'll make that update.